src/lib/database/migrations/1594413973851-V01_MigrateClientStorage.ts

```diff
@@ -17,7 +16,14 @@ export class V01MigrateClientStorage1594413973851 implements MigrationInterface
 			new Table({
 				name: 'client',
 				columns: [
-					new TableColumn({ name: 'id', type: 'varchar', length: '19', default: `'${CLIENT_ID}'`, isNullable: false, isPrimary: true }),
+					new TableColumn({
```

Comment:
How do migrations get this?

Reply:
I added an `import '#root/config';` to the `database.config.ts` so envs are loaded before migrations.